src/util/logger.ts

```diff
@@ -29,73 +29,48 @@ export function setLogLevel(l: LogLevel): void {
 }
 
 export const logger = {
-  trivial: (message: string): void => {
+  trivial: (...messages: unknown[]): void => {
     if (level > LogLevel.Trivial) {
       return;
     }
 
-    if (typeof console != 'undefined') {
-      console.log(`YORKIE T: ${message}`);
-    }
```

Comment:
Wouldn't we need code to check console availability? I don't know well about browser compatibility, but I've seen some other libraries check it.
 - [firebase-js-sdk](https://github.com/firebase/firebase-js-sdk/blob/9611fd739b9ab4b9293c16a3944b68ff997d6b08/packages/util/src/subscribe.ts#L238)
 - [caniuse console.log](https://caniuse.com/?search=console.log)

Reply:
If I only looked at [mdn console.error](https://developer.mozilla.org/en-US/docs/Web/API/Console/error), 
it was confirmed that most of them are supported and I did as above. (diff caniuse It's weird..) 


[In Stackoverflow](https://stackoverflow.com/questions/3326650/console-is-undefined-error-for-internet-explorer) that it may not exist when it is a sub-browser when searching (IE8)

Should we support IE8..?

Reply:
I haven't thought about the range of browser support yet. I think browser compatibility issues for `console.log` are outside the scope of this PR. How about doing the compatibility issue in another PR?

Reply:
@hackerwins I will do that :)